spec/matrix.yml

```diff
@@ -1,20 +1,20 @@
 ---
 OS:
-  - amazonlinux-1-x64
-  - centos-6-x64
-  - centos-7-x64
-  - debian-8-x64
-  - debian-9-x64
-  - oracle-6-x64
-  - oracle-7-x64
-  - ubuntu-server-1404-x64
-  - ubuntu-server-1604-x64
+- amazonlinux-1-x64
```

Comment:
I'm quite sure this file isn't used anymore and can be deleted.

Reply:
Maybe,  But this is not Subject, of this PR.